pkg/clusterregistry/options/options.go

```diff
@@ -20,59 +20,56 @@ package options
 import (
 	"time"
 
+	"k8s.io/apiserver/pkg/server"
 	genericoptions "k8s.io/apiserver/pkg/server/options"
 	"k8s.io/apiserver/pkg/storage/storagebackend"
 
 	"github.com/spf13/pflag"
 )
 
+type OptionsGetter interface {
```

Comment:
Maybe "ClusterRegistryOptions" or just "Options"? It seems like more than simply a getter, it also applies and validates